src/services/tags/create-tag.test.js

```diff
@@ -0,0 +1,44 @@
+const {expect} = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require("../../db")
+const createTag = require("./create-tag")
+
+
+describe('Create tag', () => {
```

Comment:
So let's always try to ensure we're testing the scenarios the code will present or presents

if it throws an error in a certain scenario, it should be part of the test cases for us to make sure it works as expected